Extensions/show_originals.js

```diff
@@ -65,8 +65,10 @@ XKit.extensions.show_originals = new Object({
 
 		XKit.extensions.show_originals.update_button();
 
-		$("#xshow_originals_ul").click(function() {
+		$("#xshoworiginals_button").click(function() {
```

Comment:
Oh, that was to point the handler at the text under the header, instead of the whole ul. Makes it more consistent with other extensions.

Seems I forgot to remove the inline onclick here though.